pipelines/skylab/multiome/atac.wdl

```diff
@@ -76,7 +97,10 @@ workflow ATAC {
         read1_fastq = TrimAdapters.fastq_trimmed_adapter_output_read1,
         read3_fastq = TrimAdapters.fastq_trimmed_adapter_output_read3,
         tar_bwa_reference = tar_bwa_reference,
-        output_base_name = input_id
+        output_base_name = input_id,
+        nthreads = num_cpus_per_node_bwa, 
```

Comment:
We're not passing all the machine info from the parameters here which makes me think we probably can reduce the number of parameters used as input

Reply:
Which machine info are you referring to?